styles.css

```
@@ -89,3 +91,307 @@ nav:hover {
+}
+
+.card2 {
+  background-color: #fff;
+  border: 1px white;
+  border-radius: 16px;
+  padding-left: 5px;
+  height: 620px;
+  width: 327px;
+  margin-top: 0;
+}
+
+.card3 {
+  background-color: #fff;
+  border: 1px white;
+  border-radius: 16px;
+  padding-left: 5px;
+  height: 580px;
+  width: 327px;
+  margin-top: 80px;
+}
+
+.card4 {
+  background-color: #fff;
+  border: 1px white;
+  border-radius: 16px;
+  margin-bottom: 80px;
+  height: 630px;
+  width: 327px;
+  margin-top: 80px;
+}
+
+.word-title {
+  font-family: 'Poppins', sans-serif;
+  padding: 16px;
+}
+
+.w-title {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+}
+
+.name {
+  font-style: bold;
+  font-weight: 800;
+  font-size: 32px;
+  color: #172b4d;
+  vertical-align: top;
+  line-height: 44px;
+  margin: -3px;
+}
+
+.canop {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  align-items: baseline;
+  align-self: flex-start;
+}
+
+.canop li:nth-child(n) {
+  margin-top: 17px;
+}
+
+.languages li:nth-child(n) {
+  border: 1px;
+  border-radius: 8px;
+  padding: 5px 12px 5px 12px;
+  background-color: #ebebff;
+  color: #6070ff;
+  letter-spacing: 3%;
+  font-size: 12px;
+  text-emphasis-style: small;
+  text-align: center;
+  padding-top: 10px;
+}
+
+.start li:nth-child(2) {
+  color: #acb7c3;
+}
+
+.lang-icons li:nth-child(n) {
+  margin-top: 10px;
+  padding: 30px;
+  border-radius: 8px;
+  background-color: #f7f7f9;
+}
+
+.frame li:nth-child(1) {
+  color: #000;
+  text-align: left;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.frame li:nth-child(2) {
+  color: #acb7c3;
+  width: 2px;
+  padding-right: 40px;
+  margin-bottom: 20px;
+}
+
+.skills li:nth-child(1) {
+  color: #000;
+  text-align: left;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+  padding-right: 10px;
+}
+
+.skills li:nth-child(2) {
+  padding: 0;
+  color: #acb7c3;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.w-title ul li:first-child {
+  list-style: none;
+  text-align: left;
+  font-size: 13px;
+  font-weight: 600;
+  color: #344563;
+}
+
+.w-title ul li:nth-child(n + 2) {
+  color: #7a869a;
+  font-size: 13px;
+  font-family: 'Poppins', sans-serif;
+  line-height: 16px;
+  font-weight: 900;
+  list-style-image: url(./images/dot1.png);
+  margin-left: 20px;
+}
+
+.p-text {
+  font-size: 15px;
+  font-weight: 400;
+  font-style: 'Poppins', sans-serif;
+  color: #344563;
+  vertical-align: text-top;
+  line-height: 44px;
+  margin: 0;
+}
+
+.word {
+  width: 295px;
+  height: 76px;
+  line-height: 24px;
+}
+
+.languages {
+  display: flex;
+  align-self: flex-start;
+  padding-left: 0;
+  gap: 10px;
+  margin-top: 10px;
+}
+
+.see {
+  border: 2px solid #dfe1e6;
+  border-radius: 8px;
+  background-color: #fff;
+  font-size: 17px;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  color: #396df2;
+  padding: 10px 10px 20px 20px;
```

[Optional]

- [ ] The padding option starts from the top and continues clockwise (top, right, bottom, left). You should update this option to make your padding equal on all axis. 
```suggestion
  padding: 10px 20px 10px 20px;
```

---

Thank you so much @sinansevgi. Let me make the necessary changes then I merge it.